.python-version

```diff
@@ -0,0 +1,2 @@
+flask_wtf
+3.9.11
```

Comment:
ooc, how did we pick this version specifically?

Reply:
random -- just a version i had installed locally.

in theory, this library should support multiple python version.

in practice, this hopefully won't matter once we delete this fork